libs/src/index.js

```diff
@@ -1,5 +1,6 @@
 const packageJSON = require('../package.json')
 
+const { AnchorAudiusData } = require('./services/anchorAudiusData/index')
```

Comment:
nit but maybe we can call this `solanaAudiusData` to be in line with other refs below? or `anchorWeb3Manager`?

Reply:
not anchor web3manager, but it's not a web3 manager - it just leverages the solanaweb3manager.
Maybe solanaAudiusData?
Since it's effectively replacing the `dataContracts` (poa data contracts service in libs), I think solanaAudiusData is fine - There's no need to anchor in the name